Wiki.py

```diff
@@ -0,0 +1,44 @@
+import wikipedia
+from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
+
+def wikiInfo():
+    cat = wikipedia.page("Cat")
+    dog = wikipedia.page("Dog")
+
+    catInfo = cat.content
+    dogInfo = dog.content
+
+    return catInfo, dogInfo
+
+def sent(info):
+    analyzer = SentimentIntensityAnalyzer()
+    return analyzer.polarity_scores(info)
+
+def topTen(info):
+    infoList = info.split()
+
+    stripped = []
+    for word in infoList:
+        stripped.append(word.strip("/.,?'':;[]"")(%"))
+
+    hist = dict()
+    for c in stripped:
+        hist[c] = hist.get(c, 0) + 1
+
+    listHist = []
+    for key in hist:
+        listHist.append((key, hist[key]))
+
+    return sorted(listHist, key=lambda words: words[1], reverse=True)
+
+
+    unwanted = ['a', 'an', 'be', 'and', 'are', 'from', 'for', 'the', 'they',
+    'their', "they're", 'then', 'them', 'is', 'if', 'of', 'with']
+
```

Comment:
```unwanted = ['a', 'an', 'be', 'and', 'are', 'from', 'for', 'the', 'they', 'their', "they're", 'then', 'them', 'is', 'if', 'of', 'with']```
This code is unnecessary. This snippet of code comes after return statement, after which a function terminates. I don't know if this is something that you just forgot to delete or an unintended bug, but this part of code never gets executed.